src/WebJobs.Script.Scaling/AppServiceSettings.cs

```diff
@@ -249,6 +250,28 @@ public static string LockHandleId
             }
         }
 
+        /// <summary>
+        /// Gets or sets a runtime encryption key
+        /// </summary>
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        public static byte[] RuntimeEncryptionKey
+        {
+            get
+            {
+                if (_runtimeEncryptionKey == null)
+                {
+                    _runtimeEncryptionKey = Convert.FromBase64String(Environment.GetEnvironmentVariable("WEBSITE_ENCRYPTION_KEY"));
```

Comment:
This will throw in environments where that variable is not defined, correct? Might good to handle those cases and either log or throw something meaningful so the issue is clear.

Reply:
Fixed.